bundle/manifests/external-dns-operator.clusterserviceversion.yaml

```diff
@@ -436,13 +436,14 @@ spec:
                 - --externaldns-image=$(RELATED_IMAGE_EXTERNAL_DNS)
                 - --trusted-ca-configmap=$(TRUSTED_CA_CONFIGMAP_NAME)
                 - --leader-elect
+                - --webhook-disable-http2
                 env:
                 - name: OPERATOR_NAMESPACE
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
                 - name: RELATED_IMAGE_EXTERNAL_DNS
-                  value: quay.io/openshift/origin-external-dns@sha256:419ae8594ba87f197c49e298ecec85e185ed29e8fd80e874351415c7ce69381e
+                  value: quay.io/external-dns-operator/external-dns@sha256:2ae9c9dc9a528443dc8d4edfca2b681a7376eff706f3b52feda763f6c4e25f07
```

Comment:
So this is a direct result of https://github.com/openshift/release/pull/46634/files#diff-a99f743ad0309322b74e5b90fbb6779685260c4ea5d77c17fc5bce861a2f21f7?

Could you help me understand the context of this old image repo: `quay.io/openshift/origin-external-dns` . Is that some sort of default image? How does that differs from quay?